frameworks/FSharp/giraffe/benchmark_config.json

```diff
@@ -0,0 +1,47 @@
+{
+  "framework": "giraffe",
+  "tests": [
+    {
+      "default": {
+        "plaintext_url": "/plaintext",
+        "json_url": "/json",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "fullstack",
+        "database": "None",
+        "framework": "giraffe",
+        "language": "F#",
+        "orm": "Raw",
+        "platform": ".NET",
+        "flavor": "CoreCLR",
+        "webserver": "Kestrel",
+        "os": "Linux",
+        "database_os": "Linux",
+        "display_name": "Giraffe",
+        "notes": "",
+        "versus": "aspcore"
+      }
+    },
+    {
+      "jsonutf8": {
+        "plaintext_url": "/plaintext",
```

Comment:
nit: Don't need plaintext entry for this one as nothing is different for plaintext, so no reason to test it twice